Rakefile

```diff
@@ -231,7 +307,8 @@ def create_directory_bundle(target, prefix = nil)
   sh "cp -pR %s %s"                  % [vendor_dir, lib_dir]
   sh "cp %s Gemfile Gemfile.lock %s" % [spec_path, dest_vendor_dir]
 
-  GEMSPEC.require_paths.each do |path|
+
+  GEMSPEC.require_paths.select { |path| path !~ /.rbenv/ }.each do |path|
```

Comment:
For me, some of the `require_paths` values were being served out of `~/.rbenv/`. I only intend to symlink through the local paths used in this gem, not any system level paths that also manage to show up in `GEMSPEC.require_paths`.

Any of this hackery is there to let the `gemspec` drive how dependencies are calculated when building the standalone Traveling Ruby bundle dir.